packages/gasket-plugin-nextjs/README.md

```diff
@@ -44,7 +44,7 @@ you would set for [custom configurations][next.config] or using Next.js plugins.
 For general Webpack configurations, it is recommend to utilize features of the
 Gasket [webpack plugin].
 
-#### Example configuration
+### Example configuration
```

Comment:
Keep example sections at level 4 heading per the [markdown contrib guide](https://github.com/godaddy/gasket/blob/main/CONTRIBUTING.md#markdown-documentation) for consistency and to avoid crowding the gasket.dev sidebar nav.